chart/templates/_helpers.tpl

```diff
@@ -58,16 +63,19 @@ helm.sh/chart: {{ include "pet-battle-tournament.chart" . }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- end }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
+app.kubernetes.io/part-of: petbattleworld
+{{ include "pet-battle-tournament.stateLabels" . }}
 {{- end -}}
 
 {{/*
 Selector labels
 */}}
+
+
 {{- define "pet-battle-tournament.selectorLabels" -}}
 app.kubernetes.io/name: {{ include "pet-battle-tournament.name" . }}
 app.kubernetes.io/component: {{ include "pet-battle-tournament.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
-deploymentconfig: {{ include "pet-battle-tournament.fullname" . }}
```

Comment:
There is some control flow in your `values.yaml` for DC on or off. Will this break the things if I now select DC on?

Reply:
yea I couldn't figure out how to do not set in helm e.g. if DC is true then ignore the deployement and vice veras